src/api/bitswap/stat.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const moduleConfig = require('../../module-config')
+
+module.exports = (arg) => {
+  const send = moduleConfig(arg)
```

Comment:
The main problem of doing this step at `index.js` level is that will lose the capability of require individual sub-modules (for example, `require('ipfs-api/bitswap/unwant')(config)`)

The `moduleConfig` already checks if the `arg` is a `send` instance, or if is necessary to init one.